templates/telco/osm/onboarding-network-functions.html

```diff
@@ -0,0 +1,205 @@
+      <h2>Day 2</h2>
+    </div>
+    <div class="col-6">
+      <ul>
+        <li>Reconfigure when needed after the service is running</li>
+        <li>Monitor the specific metrics for the infrastructure</li>
+        <li>Scale on the basis of monitoring analysis</li>
+        <li>Operate to enable closed-loop automation</li>
+      </ul>
+    </div>
+  </div>
+  <br />
+  <br />
+  <div class="row">
+    <div class="col-12 u-vertically-center u-hide--small">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/e8f2394f-Onboarding+2.svg",
+        alt="",
+        width="1000",
+        height="180",
+        hi_def=True,
+        loading="auto"
+        ) | safe
+      }}
+    </div>
+  </div>
+</section>
+
+<section class="p-strip is-bordered">
+  <div class="row u-vertically-center">
+    <div class="col-12">
+      <h2>Lifecycle management</h2>
+      <p>The goal of onboarding is to create a package which can fulfil the stages to manage the lifecycle of network services. The package should  include network function instantiation, scaling in/out, healing or recovery, updating, operating and termination.</p>
+    </div>
+		<hr class="p-separator" />
+		<div class="col-12 u-hide--small u-align--center">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/68971b5f-Onboarding+3.svg",
+        alt="",
+        width="1000",
+        height="500",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
+    </div>
+  </div>
+</section>
+<hr />
+<section class="p-strip is-bordered">
+  <div class="row">
+    <div class="col-12">
+			<h2>VNF onboarding in OSM Hackfests</h2>
+      <p>Canonical regularly participates in the <a href="https://osm.etsi.org/wikipub/index.php/Past_OSM_Hackfests" class="p-link--external">events</a> organized by the OSM community and ETSI’s Centre for Testing and Interoperability. For events like Hackfests and NFV Plugtests Canonical also offers a partner cloud where multiple VNF vendors can deploy and test the interoperability of their Telco network functions. It is also available if VNF vendors want to test their network functions individually before migrating to production environments.</p>
+      <h4>52 Participants, More than 100 Network Functions onboarded</h4>
+    </div>
+    <div class="col-12">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/a1afcfd0-V.jpg",
+        alt="",
+        width="681",
+        height="409",
+        hi_def=True,
+        loading="lazy",
+        attrs={"class": "p-image--bordered"},
+        ) | safe
+      }}
+      <p>
+        <a href="https://charmed-osm.com/osm-hackfests" class="p-link--external">
+          Discover our contributions&nbsp;&rsaquo;
+        </a>
+      </p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--suru-accent is-dark">
+  <div class="row">
+    <div class="col-12">
+      <div class="p-heading-icon--small">
+        <h4 id="get-in-touch">Tell us about your network function</h4>
```

Comment:
This hash is used by the dynamic forms which will become confusing. Can you use a different hash please.

Reply:
Updated to #tell-us